techniques/system/rudderWebapp/1.0/integrity-check.cf

```diff
@@ -0,0 +1,64 @@
+####################################################################################
+# Copyright 2021 Normation SAS
+####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+bundle agent system_root_integrity_check(trackingkey, technique_name)
```

Comment:
you could fetch the values from bundle system_rudder_webapp, wouldn't it be better ?

Reply:
I believe that we should avoid when possible to rely on data taken from other resources.